app/squac/settings.py

```diff
@@ -7,7 +7,6 @@
 import requests
 from requests.exceptions import RequestException, MissingSchema
 import os
-from squac.cronjobs import CRONJOBS
```

Comment:
I thought this needed to be included in settings.py in order to be added with "python app/manage.py crontab add" in the after-install script?
https://pypi.org/project/django-crontab/

Reply:
that was merge cruft, you are correct